### DISCUSSION TITLE
Administrator Launcher stucks for several mintues at "verifying from cache: ..."

### DISCUSSION THREAD

**Discussion by valentina9214** [nextgenhealthcare/connect#5758] [Q&A]
Hi,
I am using Mirth with OpenJDK 19. I am using the Administrator Launcher and the launcher hangs for several minutes at "verifying from cache: ...".

What could be the cause?
What can I do to make the launcher launched faster?

Thanks a lot

**Comment by SanderDB** [comment]
Install latest Administrator Launcher (1.3.1 currently).
Also read https://forums.mirthproject.io/forum/mirth-connect/support/179012-administrator-launcher-1-2-0-stucks-5-mintues-at-verifying-lib

**Comment by valentina9214** [comment]
Thanks! I install the latest version!
I think it is a problem due to the internet network or proxy.
Is there a way to make the launcher skip the proxy check? 
I try to set -Djava.net.useSystemProxies=true in the configuration of Launcher but it doesn't work!

**Reply by SanderDB** [reply]
Please read the link I provided in my previous post. It states modifying the hosts file makes the launcher 'skip' the OCSP verify HTTP requests...
